openmdao/core/problem.py

```diff
@@ -277,7 +279,7 @@ def is_local(self, name):
         bool
             True if the named system or variable is local to this process.
         """
-        if self._setup_status < 1:
+        if not hasattr(self, '_metadata'):
```

Comment:
Would probably be better to set self._metadata = None in the `Problem.__init__` method and just check if it's None here.  I should have done that when I added it but I forgot.